### ISSUE TITLE
boxee+box freezes after exiting navi-x...

### ISSUE THREAD

**Issue by crazyscow** [boxeehacks/boxeehack#521]
Hey all,

I just wanted to report this bug. Once i set navi-x to the home screen and access it, everything works great, but when i exit it, the boxee box just freezes and i need to manual restart it.

anybody else having this issue?
running the latest boxee+ (1.4.6)

tried reinstalling boxee+ but did not help. other then that great piece of Software add on!!

**Comment by tomhs** [comment]
mine too

**Comment by SBMongoos** [comment]
Yes, I have this same issue. Verify annoying as the Boxee hangs a lot and you need to go to the device and hold the power down. Then power back up. Running latest version but also happened with previous version as well.
